examples/30_extended/fetch_runtimes_tutorial.py

```diff
@@ -130,26 +160,25 @@ def print_compare_runtimes(measures):
 # larger datasets.
 print_compare_runtimes(measures)
 
+####################################################################################
+# Running the same task with a different parallel backend. Joblib provides multiple
+# backends: {`loky` (default), `multiprocessing`, `dask`, `threading`, `sequential`}.
+# The backend can be explicitly set using a joblib context manager. The behaviour of
+# the job distribution can change and therefore the scale of runtimes recorded too.
+
+with parallel_backend(backend="multiprocessing", n_jobs=-1):
```

Comment:
Is the `-1` intentional? I think it parallelizes the outer fold and therefore runs sequentially on the inner fold, making the CPU and wallclock time identical.

Reply:
The `-1` was intentional.

What would `inner folds` imply for this case? My understanding was that OpenML's parallel run would be running multiple RFs on the different OpenML data splits, while each RF fit would be distributing its trees too?

On my machine, the runtimes don't look `identical`, in fact quite erratic.
![Screenshot from 2021-04-08 15-45-06](https://user-images.githubusercontent.com/3191233/114038210-f3989b80-9881-11eb-8dfc-3a3b7f04f0bf.png)

Based on the analysis we did earlier, the first table shows that the inner context does change into a `threading` backend, but somehow the Case 2 table suggests that speed-up does happen? 
![multiprocessing-full](https://user-images.githubusercontent.com/3191233/114038691-5ee26d80-9882-11eb-87e3-63f9df0c7c37.png)

What do you suggest I do here?
To be honest, the use of a context manager and the resulting behaviour does confuse me.

Reply:
So if you do threading (because of the fallback), jobs which leave the GIL (such as the decision tree) can still run in parallel...super confusing. I guess we need to add another case? Do you know of an algorithm that does neither use blas nor the joblib? A linear SVM?